fractions/microprofile/microprofile-jwt/src/main/java/org/wildfly/swarm/microprofile/jwtauth/runtime/MPJWTAuthExtensionArchivePreparer.java

```diff
@@ -92,16 +97,19 @@ public void process() throws Exception {
                 realm = realmNameProp.asString();
             }
             // Set the web.xml login-config auth-method and jboss-web.xml security domain
-            if (authMethod != null) {
-                WebXmlAsset webXml = war.findWebXmlAsset();
-                webXml.setLoginConfig(authMethod.asString(), realm);
```

Comment:
Removing this means it will no longer be possible to set a different auth method than `MP-JWT`. Which was probably never intended, but it seems it was possible. Not sure about consequences.

Reply:
@Ladicek yes, the documentation for this annotation says that more than `MP-JWT` is possible but within the scope of `microprofile-jwt` only `MP-JWT` matters, the other standard methods like `Basic` etc are not really relevant